Python/Video to PDF(CLI)/video_to_pdf.py

```diff
@@ -0,0 +1,78 @@
+from moviepy.editor import VideoFileClip
+from moviepy.editor import AudioFileClip
+from fpdf import FPDF
+import threading
+import speech_recognition as sr
+import os
+
+#variables
+video_clip = ''
+audio_clip = ''
+
+def audio_to_pdf():
+    '''
+    Function to convert audio to pdf:
+    Here, we first extract the audio from the selected video and store it into
+    'my_audio.wav'. Then, using the speech recognition module and Google speech API,
+    we convert this audio to text and write it into a file named 'my_text.txt'.
+    This text file is then passed to text_to_pdf function to convert it into a pdf and at
+    the end the audio and text files are removed using the OS module as we don't need them anymore.
```

Comment:
You don't explain how your code works in a docstring. Just mention the function params, return type and the purpose (which you already pitched)